server/createData.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
+const faker = require('faker');
+
+function create() {
+  const result = [];
+  const photos = ['https://s3-us-west-1.amazonaws.com/homes-pic/0.jpg',
```

Comment:
I will suggest creating another file to store the URL of images. (ex. imageURL.js)
And in createData.js(this file) just simply import imageURL.js( var image = require('./imageURL.js'))
My thought is it will be easier for you to organize URLs in the future. For example, when you want to add another 30 url of photos, it might be hard to remember you stores the urls in createData.js

Reply:
Thanks! Added file with images `images.js`.